wiki/markdown/curl-examples.md

```diff
@@ -172,21 +274,64 @@ Note: It is not possible to add the `backendurl` parameter as a JSON parameter.
     </tr>
 </table>
 
-## Curl Examples
-A curl call with the command: `curl -X GET http://localhost:8080/auth` Gives the response data `{"security":false}`. The `security` parameter would be true if LDAP was enabled.
+For these endpoints to be reachable the Eiffel Intelligence back-end [needs to be configured](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties) with `testaggregated.enabled: true`.
+The below command would result in a json response of `{"status":true}` if this functionality is enabled.
+
+    curl -X GET -H "Content-type: application/json" localhost:8080/rules/rule-check/testRulePageEnabled
+
+
+Example curl commands to these endpoints [can be found here](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/Running-rules-on-objects.md)
+
+## <a id="subscriptions" />/subscriptions
+
+<table>
+    <tr>
+        <th>Context Path</th>
+        <th>Type</th>
+        <th>Explanation</th>
+    </tr>
+    <tr>
+        <td>/subscriptions</td>
+        <td>GET</td>
+        <td>Fetches all available subscriptions from the back-end</td>
+    </tr>
+    <tr>
+        <td>/subscriptions</td>
+        <td>POST</td>
+        <td>Request to add one (or more) new subscription(s) included in JSON format</td>
+    </tr>
+    <tr>
+        <td>/subscriptions</td>
+        <td>PUT</td>
+        <td>Request to update one (or more) subscription object(s) included in JSON format</td>
+    </tr>
+    <tr>
+        <td>/subscriptions/*subscription name*</td>
+        <td>GET</td>
+        <td>Fetches information about a single (or multiple) subscription(s) from the back-end</td>
+    </tr>
+    <tr>
+        <td>/subscriptions/*subscription name*</td>
+        <td>DELETE</td>
+        <td>Deletes a single (or multiple) subscription(s) from the back-end</td>
```

Comment:
An example showing how you delete multiple would be good to add.

Reply:
Further examples are shown in the back-end docs.

Reply:
I'm linking to the back-end docs which contains a lot more examples. I think that should be enough.